packages/kubernetes-1.15/kubelet-config

```diff
@@ -30,4 +30,4 @@ serverTLSBootstrap: true
 configMapAndSecretChangeDetectionStrategy: Cache
 tlsCipherSuites:
 - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
-MaxPods: {{default 110 settings.kubernetes.max-pods}}
+maxPods: {{default 110 settings.kubernetes.max-pods}}
```

Comment:
If we're changing it for 1.15 and 1.16 too, would you please updating Testing Done to include those?

Reply:
Updated PR description to include testing information for 1.15, 1.16, 1.17